src/js/_enqueues/wp/heartbeat.js

```diff
@@ -223,16 +223,33 @@
 				settings.checkFocusTimer = window.setInterval( checkFocus, 10000 );
 			}
 
-			$(window).on( 'unload.wp-heartbeat', function() {
+			$(window).on( 'pagehide.wp-heartbeat', function() {
 				// Don't connect anymore.
-				settings.suspend = true;
+				suspend();
 
 				// Abort the last request if not completed.
 				if ( settings.xhr && settings.xhr.readyState !== 4 ) {
 					settings.xhr.abort();
 				}
 			});
 
+			$(window).on(
+				'pageshow.wp-heartbeat',
+				/**
+				 * Handles pageshow event.
+				 *
+				 * @param {jQuery.Event} event
+				 * @param {PageTransitionEvent} event.originalEvent
+				 */
+				function ( event ) {
+					if ( event.originalEvent.persisted ) {
+
+						// Continue connecting.
+						resume();
+					}
+				}
+			);
```

Comment:
I'm not seeing this code firing.

Reply:
OK, I am seeing it fire. Apparently I have a browser extension which is disabling bfcache, but if I try in an incognito window then `event.originalEvent.persisted` is `true`.